### ISSUE TITLE
error

### ISSUE THREAD

**Issue by lcxsnow** [deepakbaby/isegan#7]
I'm running "run_isegan.py" but it appears error "Error polling for event status: failed to query event: CUDA_ERROR_ILLEGAL_ADDRESS: an illegal memory access was encountered".
My tensorflow is 1.13.1, cuda is 10.0.1, cudnn is 7.6.5 and keras is 2.2.4.

**Comment by deepakbaby** [comment]
Hi @lcxsnow,
I dont think it is an error with the code itself. It seems it is an error with your GPU. https://stackoverflow.com/questions/60771063/tensorflow-reported-cuda-error-illegal-address-bug-while-train-yolo

**Comment by lcxsnow** [comment]
I upgraded TensorFlow into 1.14 and closed other programs when training. Now it's fine.

**Comment by lcxsnow** [comment]
In run_isegan.py or models.py, Gtanh isn't declared. Is it important to declared 'opts[Gtanh] = True '? Or just comment the  
"  if opts["Gtanh"]:
        dec_out = Activation('tanh')(dec_out) " in models.py "

**Comment by lcxsnow** [comment]
How to predict and see the training accuracy?

**Comment by deepakbaby** [comment]
Please comment it tanh part. Adding tanh did not help, I forgot to remove it. Thanks. 

Regarding training accuracy, what do you mean by accuracy here? The models are trained to minimise the GAN loss and also additional L1 loss. The training procedure displays those losses.

**Comment by lcxsnow** [comment]
After testing,  there are several clean.wav in test_result_epoch(x) folder. I mean that "accuracy" equals those clean .wav compare with clean_train_wav. How many files are the same as clean_train_wav or how much similar with clean_train_wav?

**Comment by deepakbaby** [comment]
Since this is a speech enhancement system, the performance can be measured by some metric such as SDR or so,. You can check the paper to see the list of evaluation metrics used.

**Comment by lcxsnow** [comment]
Ok. Thanks.

**Comment by lcxsnow** [comment]
How to calculate STOI, PESQ, MOS, SNR, CD, and LLD?
